monitoring-custom-metrics/k8s/service-8081.yaml

```diff
@@ -0,0 +1,16 @@
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: sample-metrics-8081
+  labels:
+    k8s-app: metrics
+    example: monitoring-custom-metrics
+  namespace: default
+spec:
+  selector:
```

Comment:
There are two services with the same selector. Why don't you merge them together?

Reply:
As discussed, will keep it as is.